resources/views/auth/application/app.blade.php

```diff
@@ -49,34 +53,35 @@
             @endforeach
         </div>
     </div>
-    @if($user->application->status == App\Models\ApplicationForm::STATUS_IN_PROGRESS)
+    @if(!$user->application->submitted)
         <nav class="nav-extended">
             <div class="nav-content">
                 <ul class="tabs tabs-transparent">
                     <li class="tab">
                         <a href="{{ route('application', ['page' => 'personal']) }}"
-                           class="@yield('personal-active')">Általános</a></li>
+                           class="{{request()->get('page') == 'personal' ? "active" : ""}}">Általános</a></li>
                     <li class="tab">
                         <a href="{{ route('application', ['page' => 'educational']) }}"
-                           class="@yield('educational-active')">Tanulmányok</a></li>
+                           class="{{request()->get('page') == 'educational' ? "active" : ""}}">Tanulmányok</a></li>
                     <li class="tab">
                         <a href="{{ route('application', ['page' => 'questions']) }}"
-                           class="@yield('questions-active')">Egyéb kérdések</a></li>
+                           class="{{request()->get('page') == 'questions' ? "active" : ""}}">Egyéb kérdések</a></li>
                     <li class="tab">
                         <a href="{{ route('application', ['page' => 'files']) }}"
-                           class="@yield('files-active')">Fájlok</a></li>
-                    <li class="tab right">
-                        <a href="{{ route('application', ['page' => 'submit']) }}"
-                           class="@yield('finalize-active')">Ellenőrzés és véglegesítés</a></li>
+                           class="{{request()->get('page') == 'files' ? "active" : ""}}">Fájlok</a></li>
                 </ul>
             </div>
         </nav>
         @yield('form')
     @else
         @include('auth.application.application', ['user' => $user])
     @endif
-    @if($user->application->status != App\Models\ApplicationForm::STATUS_IN_PROGRESS && isset($user->internetAccess))
+    @if($user->application->submitted)
         @include('network.internet.wifi_password', ['internet_access' => $user->internetAccess, 'application' => true])
     @endif
 
+    @if(request()->get('page') != 'submit')
+    <x-input.button href="{{ route('application', ['page' => 'submit']) }}"
+       class="right coli blue" text="Ellenőrzés és véglegesítés" />
+    @endif
```

Comment:
For me, the cookie disclaimer covers the button. I think we should simply left the submission option where it originally was; I would look for it there.

Reply:
That was also hidden on some special cases (on mobiles). It was not clear that the navbar is scrollable.

Reply:
Do you think the cookie disclaimer would be an issue?

Reply:
I see… we can move it to a button then; just let’s write some hint on where it is. E.g. _”Miután minden szükséges mezőt megválaszolt és fájlt feltöltött, kérjük, véglegesítse a jelentkezését **a lap alján levő gombra kattintva**.”_